source/blog/2017-06-06-using-kubeadm-with-cri-o.html.md

```diff
@@ -0,0 +1,312 @@
+
+Setting up the Master
+-
+
+Start by logging into the `fedora.vm` machine which will be the master node:
+
+```sh
+fedora.vm $ cat <<EOF > /etc/yum.repos.d/kubernetes.repo
+[kubernetes]
+name=Kubernetes
+baseurl=https://packages.cloud.google.com/yum/repos/kubernetes-el7-x86_64
+enabled=1
+gpgcheck=1
+repo_gpgcheck=1
+gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg
+        https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
+EOF
+fedora.vm $ dnf install -y kubelet kubeadm kubernetes-cni
+fedora.vm $ systemctl restart crio
+fedora.vm $ systemctl enable kubelet && systemctl start kubelet
+```
+
+Run `kubeadm` to install the master node:
+
+```sh
+fedora.vm $ kubeadm init --pod-network-cidr=10.244.0.0/16 --skip-preflight-checks
+[kubeadm] WARNING: kubeadm is in beta, please do not use it for production clusters.
+[init] Using Kubernetes version: v1.6.4
+[init] Using Authorization mode: RBAC
+[preflight] Skipping pre-flight checks
+[certificates] Generated CA certificate and key.
+[certificates] Generated API server certificate and key.
+[certificates] API Server serving cert is signed for DNS names [kubeadm-master kubernetes kubernetes.default kubernetes.default.svc kubernetes.default.svc.cluster.local] and IPs [10.96.0.1 10.138.0.4]
+[certificates] Generated API server kubelet client certificate and key.
+[certificates] Generated service account token signing key and public key.
+[certificates] Generated front-proxy CA certificate and key.
+[certificates] Generated front-proxy client certificate and key.
+[certificates] Valid certificates and keys now exist in "/etc/kubernetes/pki"
+[kubeconfig] Wrote KubeConfig file to disk: "/etc/kubernetes/admin.conf"
+[kubeconfig] Wrote KubeConfig file to disk: "/etc/kubernetes/kubelet.conf"
+[kubeconfig] Wrote KubeConfig file to disk: "/etc/kubernetes/controller-manager.conf"
+[kubeconfig] Wrote KubeConfig file to disk: "/etc/kubernetes/scheduler.conf"
+[apiclient] Created API client, waiting for the control plane to become ready
+[apiclient] All control plane components are healthy after 16.772251 seconds
+[apiclient] Waiting for at least one node to register and become ready
+[apiclient] First node is ready after 5.002536 seconds
+[apiclient] Test deployment succeeded
+[token] Using token: <token>
+[apiconfig] Created RBAC rules
+[addons] Created essential addon: kube-proxy
+[addons] Created essential addon: kube-dns
+
+Your Kubernetes master has initialized successfully!
+
+To start using your cluster, you need to run (as a regular user):
+
+  sudo cp /etc/kubernetes/admin.conf $HOME/
+  sudo chown $(id -u):$(id -g) $HOME/admin.conf
+  export KUBECONFIG=$HOME/admin.conf
+
+You should now deploy a pod network to the cluster.
+Run "kubectl apply -f [podnetwork].yaml" with one of the options listed at:
+  http://kubernetes.io/docs/admin/addons/
```

Comment:
Yah, per IRC discussion you need a big disclaimer above so that users *don't* follow the kubeadm instructions, because not only will they not work, they'llbe hard to undo.